Backpack/src/test/java/net/skyscanner/backpack/calendar2/data/CalendarCellsLayoutTests.kt

```diff
@@ -41,12 +42,10 @@ class CalendarCellsLayoutTests {
   fun `there are leading spaces in month`() {
     testCalendarWith(CalendarSettings.Default) {
       verify {
-        assertTrue(state.cells[1] is CalendarCell.Space)
-        assertTrue(state.cells[2] is CalendarCell.Space)
-        assertTrue(state.cells[3] is CalendarCell.Space)
-        assertTrue(state.cells[4] is CalendarCell.Space)
-        assertTrue(state.cells[5] is CalendarCell.Space)
-        assertTrue(state.cells[6] is CalendarCell.Space)
+        for (i in 1..6) {
+          assertTrue(state.cells[i] is CalendarCell.Space)
+          assertEquals(CalendarSettings.Default.range.start.yearMonth(), state.cells[i].yearMonth)
```

Comment:
Does this assertion belong here? Is it related to verifying "there are leading spaces in month" ?

Reply:
Yep, this test verifies there's enough spacings inserted and the spacing have correct format.